packages/ui/src/components/Visualization/Custom/Group/CustomGroupExpanded.scss

```diff
@@ -81,6 +81,16 @@
       right: 0;
       margin: var(--pf-t--global--spacer--xs);
     }
+
+    &__autostart-icon {
+      position: absolute;
+      top: 2px;
+      right: 8px;
+      margin: var(--pf-t--global--spacer--xs);
+      color: rgb(0 0 0);
+      font-size: 16px;
+      z-index: 2;
+    }
```

Comment:
With this patch, we get the icon in black color

<img width="193" height="99" alt="Image" src="https://github.com/user-attachments/assets/27dcda72-0f29-47a5-a690-03f3841d91f5" />

I'm gonna apply this section to bring it back as white in light mode, and black in dark mode.

```suggestion
    &__autostart-icon {
      position: absolute;
      top: 2px;
      right: 8px;
      margin: var(--pf-t--global--spacer--xs);
      font-size: 16px;
      z-index: 2;

      > * {
        color: var(--pf-t--global--icon--color--on-brand--default);
      }
    }
```